client/src/components/ReviewTile.js

```diff
@@ -1,24 +1,109 @@
-import React from "react"
+import React, { useState, useEffect, setStatus } from "react"
+
+import NewReviewForm from "./NewReviewForm.js"
+import EditReviewForm from "./EditReviewForm.js"
 
 const ReviewTile = (props) => {
+  const review = props.review
+
   let upVotes = 0
   let downVotes = 0
-  
-  props.votes.forEach(vote => {
+
+  review.votes.forEach(vote => {
     if(vote.direction === "up") {
       upVotes++
     } else {
       downVotes++
     }
   })
 
-  return (
-    <div>
-      <p>Score: {props.score} out of 5</p>
-      <p>{props.body}</p>
-      <p>Upvotes: {upVotes}  Downvotes: {downVotes}</p>
-    </div>
-  )
+  const reviewId = review.id
+  const showId = review.showId
+ 
+  const handleDelete = async (event) => {
+    event.preventDefault()
+
+    try {
+      const response = await fetch(`/api/v1/shows/${showId}/reviews/${reviewId}`, {
+        method: "DELETE"
+      }).then(() => setStatus('Deleted') )
+    } catch(error) {
+      console.error(`Error in Fetch: ${error.message}`)
+    }
```

Comment:
still unsure what `setStatus` is here, and furthermore we should have full error handling and we should prefer using `async...await` syntax instead of .then for consistency.